python/cxx/unit_system.cpp

```diff
@@ -0,0 +1,10 @@
+#include <opm/parser/eclipse/Units/UnitSystem.hpp>
+
+#include "export.hpp"
+
+
+void python::common::export_UnitSystem(py::module& module)
+{
+    py::class_<UnitSystem>(module, "UnitSystem")
+        .def_property_readonly( "type", &UnitSystem::getName );
```

Comment:
I think this property should be called `name`

Reply:
please note the jenkins failure.